### PR TITLE
fix some lint errors

### PR DESCRIPTION
**Pull request by hannesa2**
I was not able to build without it

### CODE REVIEW

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  <p>
+  http://www.apache.org/licenses/LICENSE-2.0
+  <p>
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
```
Comment (xiphirx) [inline taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java]: Lets undo these changes

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java [hunk 2]
```diff
@@ -51,6 +52,7 @@
     super(findNavView ? findNavView(toolbar) : findOverflowView(toolbar), title, description);
   }
 
+  @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
```
Comment (xiphirx) [inline taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java]: Lollipop is not required for these functions.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -22,24 +21,26 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_gravity="bottom"
+            android:background="@color/colorPrimary"
```
Comment (xiphirx) [inline app/src/main/res/layout/activity_main.xml]: this should be ?attr/colorPrimary

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java (hunk 1 above)
Comment (hannesa2) [reply taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java]: Ok, lets keep this lint warning 

image
and prevent from a green sign
image

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java (hunk 2 above)
Comment (hannesa2) [reply taptargetview/src/main/java/com/getkeepsafe/taptargetview/ToolbarTapTarget.java]: image

feel free to change it to the way you prefer

**Comment by hannesa2** [comment]
I made new changes on request
